packages/pg/test/unit/client/escape-tests.js

```diff
@@ -1,65 +0,0 @@
-'use strict'
```

Comment:
deleting tests means breaking, non-backwards compatible changes.  We shouldn't do this.  For legacy reasons you should leave the methods on `client` but only document them on the main export `pg` object. They need to be properly deprecated and removed in a multi-month long release cycle.

Reply:
Works for me. For what it is worth the tests were moved from client to util, but it is still a breaking change. I wasn't sure if implementing charmander's suggestions would be applicable to this PR, since it was a breaking API change. I'll make these changes backwards compatible, thus no changes to unit tests will be needed.